src/client/game.cpp

```diff
@@ -4257,6 +4218,54 @@ void Game::updateShadows()
 	shadow->getDirectionalLight().update_frustum(camera, client, m_camera_offset_changed);
 }
 
+void Game::drawScene(ProfilerGraph *graph, RunStats *stats, f32 dtime)
+{
+	const video::SColor skycolor = this->sky->getSkyColor();
+
+	TimeTaker tt_draw("Draw scene", nullptr, PRECISION_MICRO);
+	this->driver->beginScene(true, true, skycolor);
+
+	const LocalPlayer *player = client->getEnv().getLocalPlayer();
```

Comment:
```suggestion
	const LocalPlayer *player = this->client->getEnv().getLocalPlayer();
```

Reply:
do we really want to consistently add pointless syntax to our functions?

Reply:
> do we really want to consistently add pointless syntax to our functions?

I didn't have an opinion on adding `this`, I just wanted to make sure it was done consistently if it was done. But now I see another argument against adding `this`: whether the code uses `this` or not will no longer be consistent in `game.cpp`, but will differ from function to function.